### ISSUE TITLE
Confirmation dialogs/modals should make background greyed out items inert for keyboard navigation

### ISSUE THREAD

**Issue by corneliusroemer** [loculus-project/loculus#6744]
I noticed that our confirmation dialogs are hard to use with keyboard navigation as tabbing goes through buttons in the greyed out background first.

We should somehow make the modal be in the foreground and everything else inert.

<img width="1920" height="1050" alt="Image" src="https://github.com/user-attachments/assets/65b1b554-2abf-4295-b0c5-4f28603fc6a5" />

Also the focus outline of blue buttons should be a color other than blue - it's hardly noticeable right now that the buttons are focused.
